modules/sdk/gradle-plugins-poshi-runner/src/main/java/com/liferay/gradle/plugins/poshi/runner/PoshiRunnerExtension.java

```diff
@@ -71,6 +81,7 @@ public void setVersion(String version) {
 	private Object _baseDir = "poshi-tests";
 	private String _openCVVersion = "2.4.9-0.9";
 	private final Map<String, Object> _poshiProperties = new HashMap<>();
+	private Object _poshiRunnerProperties = "poshi-runner.properties";
```

Comment:
I would call this variable `poshiPropertiesFile`. It's of type `Object`, but it will be resolved into a `File`, and also we want to match the name of the `poshiProperties` variable.